packages/attribute-registry-process/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "pagopa-interop-attribute-registry-process",
+  "version": "1.0.0",
+  "description": "PagoPA Interoperability service for attribute registry",
+  "main": "dist",
+  "type": "module",
+  "scripts": {
+    "test": "vitest",
+    "lint": "eslint . --ext .ts,.tsx",
+    "lint:autofix": "eslint . --ext .ts,.tsx --fix",
+    "format:check": "prettier --check src",
+    "format:write": "prettier --write src",
+    "start": "node --watch --no-warnings --loader ts-node/esm -r 'dotenv-flow/config' ./src/index.ts",
+    "build": "tsc && pnpm run copy-sql-files",
+    "copy-sql-files": "echo TODO",
+    "generate-model": "mkdir -p ./src/model/generated && pnpm openapi-zod-client './open-api/attribute-registry-service-spec.yml' -o './src/model/generated/api.ts'",
+    "clean-generated": "pnpm exec rm ./src/models/generated/api.ts",
+    "infra:start": "./scripts/db-start.sh",
+    "infra:stop": "./scripts/db-stop.sh",
+    "infra:destroy": "./scripts/db-destroy.sh"
```

Comment:
In the recent `infra refactor` we centralized the `infra:start/stop/destroy` script so this is no longer necessary